packages/server-render/server-sink.js

```diff
@@ -1,3 +1,6 @@
+const HEAD_REGEX = /<head[^>]*>((.|[\n\r])*)<\/head>/im
+const BODY_REGEX = /<body[^>]*>((.|[\n\r])*)<\/body>/im;
```

Comment:
If you want to do this right, you must use an actual parser to extract this HTML, like [the one we use in `server-register.js`](https://github.com/meteor/meteor/blob/d416ed913dc3a465c89d02cb2bdb07cadfa512f7/packages/server-render/server-register.js#L28-L45). HTML is not a regular language (the class of languages that regular expressions are adequate for parsing), but a context-free language.